install.md

```diff
@@ -1,240 +1,227 @@
 ```
-gcloud container clusters create $CLUSTER_NAME \
---cluster-version 1.9.7-gke.3 \
---zone us-central1-a \
---num-nodes 6
-```
-Now, add run command to add kubectl config to connect to this cluster.
-```
-gcloud container clusters get-credentials $CLUSTER_NAME -zone us-central1-a --project $PROJECT_ID
+
+If you have forgotten your project ID, you may use:
+
+```shell
+gcloud projects list
 ```
 
-### Enabling the Container Analysis API
+## Step #2: Enable Billing on your Google Cloud Project
 
-You will need to enable the Container Analysis API and enable Vulnerability Scanning in your Google Cloud Console project.
-Instructions can be found in the `Before you Begin` section of the [Getting Image Vulnerabilities](https://cloud.google.com/container-registry/docs/get-image-vulnerabilities#before_you_begin) docs.
-kritis can only inspect images hosted in projects that have both of these enabled, and have already been scanned for vulnerabilities.
+Ensure that your new project has a billing account associated to it, otherwise the requisite API's will fail to enable: https://console.cloud.google.com/billing
 
-### Creating a Container Analysis Secret
-You will need to create a Kubernetes secret, which will provide kritis with the auth required to get vulnerability information for images.
-You can create the secret through the Google Cloud Console or on the command line with gcloud.
+## Step #3: Enable the requisite API's for your Google Cloud Project
 
-#### Creating Container Analysis Secret via Command Line
-Before you start, please make sure you are running as a user with permissions to create service accounts.
+NOTE: You will need to get your Google Account whitelisted to use the Container Analysis API. Join the [Container Analysis Users Group](https://groups.google.com/forum/#!forum/containeranalysis-users) to be whitelisted. It will take 1-5 business days to approve the request. Once approved, you will need to visit following links to enable the necessary API's:
 
-First, lets define variables for your serviceaccount and glcoud project.
-```
-ACC_NAME=kritis-ca-admin
-ACC_DISP_NAME="Kritis Service Account"
-```
+*  [Enable the Container Analysis API](https://console.cloud.google.com/flows/enableapi?apiid=containeranalysis.googleapis.com&redirect=https://cloud.google.com/container-registry/docs/get-image-vulnerabilities)
+* [Enable the Kubernetes API](https://console.cloud.google.com/projectselector/kubernetes)
+* [Enable vulnerability scanning](https://console.cloud.google.com/gcr/settings)
 
-1. First create the service account
-```
-gcloud iam service-accounts create $ACC_NAME --display-name "$ACC_DISP_NAME"
-```
-This should create a service account $ACC_NAME@$PROJECT_ID.iam.gserviceaccount.com.
+For more documentation, see [Container Analysis Overview](https://cloud.google.com/container-registry/docs/container-analysis). 
+
+## Step #4: Create a cluster
+
+You may skip this step if you already have a cluster configured.
```

Comment:
the cluster needs to be version  > 1.9.2. Add  note for that.
Also, we had the version number mentioned on the `gcloud cluster create`. Did you drop that by mistake or the default is now 1.9.3?

Reply:
Added a note. The default is now 1.9.7: https://cloud.google.com/kubernetes-engine/versioning-and-upgrades